pkgs/by-name/se/seventeenlands/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  fetchPypi,
+  lib,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "seventeenlands";
+  version = "0.1.42";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-P/imV4vvyd6wgjqXzgfIAURFtFhLwX1eS8eiPl79oZk=";
+  };
+
+  # No tests
+  doCheck = false;
+
+  propagatedBuildInputs = with python3.pkgs; [
+    python-dateutil
+    requests
+    tkinter
+  ];
+
+  meta = with lib; {
+    description = "Simple client for passing relevant events from MTG Arena logs to the 17Lands REST endpoint.";
```

Comment:
```suggestion
    description = "Passing relevant events from MTG Arena logs to the 17Lands REST endpoint, also named mtga-log-client";
```

Reply:
Thanks for the suggestion. I’ve updated the description but I left the "client" part, which I think is important.